src/Components/SortDropDown/SortDropDown.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import './SortDropDown.scss'
+
+const SortDropDown = () => {
+
+  return (
+    <select onchange="selectedSubjectName()" className="dropDown">
```

Comment:
I think we could change onchange to onChange. I'm also wondering where that function is coming from and whether it should be an anonymous function or not?